packages/cache-utils/package.json

```diff
@@ -20,13 +20,15 @@
     "del": "^5.1.0",
     "get-stream": "^5.1.0",
     "global-cache-dir": "^1.0.1",
+    "globby": "^11.0.1",
```

Comment:
I think `v11` requires node 10 https://github.com/sindresorhus/globby/releases/tag/v11.0.0

Reply:
Thanks @erezrokah, good catch!
I am getting antsy about dropping support for Node 8...